src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -200,14 +200,14 @@ class KotlinPoetTest {
   @Test fun rawStrings() {
     val source = FileSpec.get(tacosPackage, TypeSpec.classBuilder("Taco")
         .addFunction(FunSpec.builder("strings")
-            .addStatement("val a = %S", "\"\n")
-            .addStatement("val b = %S", "a\"\"\"b\"\"\"\"\"\"c\n")
-            .addStatement("val c = %S", """
+            .addCode("val a = %S\n", "\"\n")
```

Comment:
What's the behavior if this were to remain `addStatement`? Forcing people to change the method they're calling isn't appealing.

Reply:
Inside a statement, every new line gets double-indented, so in this case the formatting will be weird. 

Since formatting rules for multi-line expressions in Kotlin vary, I feel like `addStatement()` doesn't make much sense as is and has to be reworked.